### DISCUSSION TITLE
NVIM NotSignedIn problem when already Authenticated

### DISCUSSION THREAD

**Discussion by febriliankr** [community/community#35862] [Copilot Conversations] [answered]
I have tried 

`Copilot auth` and `Copilot setup`, and it returns this:
![image](https://user-images.githubusercontent.com/65109548/195247589-1d99eaaf-95dc-42cb-b51d-1f040212579b.png)

Copilot status returns `enabled and online`

I have all the proper versions for NVIM, copilot.vim and nodejs, `Copilot version` returns:
```
copilot.vim 1.6.0
Neovim 0.7.2
copilot/dist/agent.js 1.50.0
Node.js 16.16.0
```

But I can't get any copilot suggestions when coding and I get this for `Copilot panel`
![image](https://user-images.githubusercontent.com/65109548/195247776-7a23ab35-b536-4c6b-b8e5-ab0f74ef13bf.png)

What should I do? Thank you in advance

**Comment by iamcjwilliams** [comment]
@febriliankr Our copilot team is investigating this issue

**Comment by tpope** [comment, marked as answer]
@febriliankr can you try `:Copilot signout`, then `:Copilot setup` again, and see if that fixes it?

**Reply by febriliankr** [reply]
Restarted my machine, now it's working 🤔 Kinda weird, mine is M1 mac if anyone's wondering

Thank you for the assistance!

**Comment by febriliankr** [comment]
Ran across this problem just now, `:Copilot signout` and `:Copilot auth` fixed it. Still an up-to-date possible solution to this problem :D
